charts/openfaas/v5.7.2/values.yaml

```diff
@@ -0,0 +1,197 @@
+functionNamespace: ""  # Multiple namespaces is not supported in Rancher
```

Comment:
I think the fuctionNS is not for multi-apps deployment but a namespace declaration of openfass functionless apps, so it should be compatible with Rancher's catalog?

Reply:
I don't think it is for multi-app deployment either, but when I tried deploying it with a separate namespace for functions I got the following error:
`Failed to install app openfaas. Error: release openfaas failed: resource's namespace openfaas-fn doesn't match the current namespace openfaas`
It seems Rancher is not creating the "openfaas-fn" namespace. Unless Rancher expects me to configure something that I missed?

Reply:
I see, thanks for the update.